extensions/VisualEditor/wikia/modules/ve/ui/styles/images/icons/preview-photo.svg

```diff
@@ -1,8 +1,8 @@
 <?xml version="1.0" encoding="iso-8859-1"?>
 <!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd">
 <svg version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
-	 width="14px" height="14px" viewBox="0 0 14 14" enable-background="new 0 0 14 14" xml:space="preserve">
-<g fill="#7f7f7f">
+	 width="24px" height="24px" viewBox="0 0 14 14" enable-background="new 0 0 14 14" xml:space="preserve">
```

Comment:
Are these modified svgs used anywhere else? If yes, we should check how they look now.